subscribers:

 camera_reading:
  topic: /camera/rgb/image_raw
  queue_size: 1

 camera_depth:
  topic: /camera/depth/image_raw
  queue_size: 1

publishers:

  detection_image:
    topic: /image_rectification/depth_image
    queue_size: 1
    latch: true

